inst/rmarkdown/templates/science/resources/template.tex

```diff
@@ -0,0 +1,240 @@
+% Use only LaTeX2e, calling the article.cls class and 12-point type.
+\documentclass[12pt]{article}
+
+% Users of the {thebibliography} environment or BibTeX should use the
+% scicite.sty package, downloadable from *Science* at
+% www.sciencemag.org/about/authors/prep/TeX_help/ .
+% This package should properly format in-text
+% reference calls and reference-list numbers.
+
+% Science template required packages
+\usepackage{scicite}
+
+\usepackage{times}
+
+% add-ons for rticles
+% based on arxiv.sty
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
+\usepackage[hidelinks]{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
```

Comment:
This is already loaded below under the table part

Reply:
Good point, removed.